storage/cloudspanner/map_storage.go

```diff
@@ -340,9 +340,7 @@ func (t *treeTX) parallelGetMerkleNodes(ctx context.Context, rev int64) func([]s
 		close(c)
 		ret := make([]*storagepb.SubtreeProto, 0, len(ids))
 		for st := range c {
-			if st != nil {
-				ret = append(ret, st)
-			}
+			ret = append(ret, st)
```

Comment:
Does this fix some bug? I can see how the upstream code could fail because of this (see line 298 getting a nil `subtree`).

Reply:
I'm getting hung requests in practice.  
I can't say for sure that this resolves the bug, but I'm happier with this style of of parallelism than the previous one. Seems like less can go wrong.

Reply:
Wrong comment -- this if just seemed unnecessary to me. All code paths that result in a nil response return an error.

Reply:
It feels like `getSubtree` call 13 lines above can return nil. This is a valid case, because some empty tiles can be missing in the storage (e.g. when the tree is initially empty).

Reply:
Fixed.